native-schema-registry/pom.xml

```diff
@@ -0,0 +1,154 @@
+<!--
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>${parent.groupId}</groupId>
+    <artifactId>native-schema-registry</artifactId>
+    <name>Native AWS Schema Registry</name>
+    <description>AWS Glue Schema Registry native shared library wrapper</description>
+    <url>https://aws.amazon.com/glue</url>
+    <version>${parent.version}</version>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>software.amazon.glue</groupId>
+        <artifactId>schema-registry-parent</artifactId>
+        <version>1.1.10</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <scm>
+        <connection>scm:git:https://github.com/aws/aws-glue-schema-registry.git</connection>
+        <developerConnection>scm:git:git@github.com:aws/aws-glue-schema-registry.git</developerConnection>
+        <url>https://github.com/awslabs/aws-glue-schema-registry.git</url>
+    </scm>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
Are we using this anywhere? The parent pom uses 1.8 for the source and target. Should we keep it consistent?

Reply:
This module will use a different Java compiler than the parent. Since this will be compiled into a shared library, the version of JDK is not significant. I wanted to use the best stable JDK version supported by GraalVM.

Reply:
Will this introduce incompatibility issue for customers using JDK8?

Reply:
It should not be any issue, this is the java version used by GraalVM and it does not affect customer. In addition, customer are not using Java at all in this case since this shared lib is built for other languages such as Python/C# etc.

Reply:
Also, the parent module shouldn't compile this sub-module as OpenJDK / Amazon Corretto won't be able to compile this. It will use API specific to GraalVM.